bundles/org.openhab.ui/web/src/components/widgets/system/oh-knob.vue

```diff
@@ -1,32 +1,24 @@
 <template>
-  <div>
-    <knob-control v-if="!config.useSliderControl" v-bind="config" :text-color="config.textColor || (($f7.data.themeOptions.dark === 'dark') ? '#ffffff' : undefined)" :value="value"
-                  @input="sendCommandDebounced($event)" @click.native.stop="sendCommandDebounced(value, true)" @touchend.native.stop="sendCommandDebounced(value, true)" />
-
-    <round-slider v-else :value="value" :min="config.min" :max="config.max" :step="config.stepSize" :radius="config.size/2"
-                  :style="`stroke-dasharray: ${(config.dottedPath) ? config.dottedPath : 0}`"
-                  :rangeColor="config.primaryColor" :pathColor="config.secondaryColor"
-                  :tooltipColor="config.textColor || (($f7.data.themeOptions.dark === 'dark') ? '#ffffff' : undefined)"
-                  :disabled="config.disabled" :width="config.strokeWidth" :line-cap="config.lineCap"
-                  :startAngle="config.startAngle" :endAngle="config.endAngle"
-                  :borderWidth="config.borderWidth" :borderColor="config.borderColor" :circleShape="config.circleShape"
-                  @input="sendCommandDebounced($event)" @click.native.stop="sendCommandDebounced(value, true)" @touchend.native.stop="sendCommandDebounced(value, true)" />
-  </div>
+  <round-slider v-bind="config" :value="value" :min="config.min" :max="config.max" :step="config.stepSize" :radius="(config.responsive) ? config.size/2 + '%' : config.size/2"
+                :style="`stroke-dasharray: ${(config.dottedPath) ? config.dottedPath : 0}`"
+                :rangeColor="config.primaryColor" :pathColor="config.secondaryColor"
+                :tooltipColor="config.textColor" mouseScrollAction="true"
+                :disabled="config.disabled" :width="config.strokeWidth" :line-cap="config.lineCap"
+                :startAngle="config.startAngle ? config.startAngle : -50" :endAngle="config.endAngle ? config.endAngle : -130"
```

Comment:
instead of all that maybe it would be better to have perhaps a `resolvedConfig` computed property and do a `v-bind`  of that, instead of ` config`? This would lookup the former properties but still fall back to the actual properties if defined.

I'd like to offer the fact that the vue-round-slider config could be fed as-is instead of having some arbitrary mappings (for example, _rangeColor_ is mapped to _primaryColor_ but it means that a user can't set the original property _rangeColor_, I think it's rather confusing).

Reply:
So you propose to v-bind a computed resolvedConfig, where this resolvedConfig is falling back to the old config if the new properties aren’t set, correct?

Reply:
Yeah, what I meant is that if a user wants to set e.g. `tooltipColor` they should be able to use either `tooltipColor` or `textColor` (the former property name) in config, probably in that order.

Reply:
That's possible with the latest commits :+1:

Reply:
From my side this is finished now.